src/components/review/ReviewsList.js

```diff
@@ -28,8 +28,7 @@ const AlbumReviews = ({ albumId }) => {
       setError('Error fetching album reviews. Please try again later.');
       setLoading(false);
     }
-  }, [albumId, user?._id]);
-
+  }, [albumId, user?.user?._id]);
```

Comment:
Per the linter; we are not using `user.user.id` inside the `useCallback` so we can go ahead and remove it so that React is not unecessarily tracking that dependency

```suggestion
  }, [albumId]);
```